### PR TITLE
fix(docker): fix DB creation in postgres container

### PR DESCRIPTION
**Pull request by cmltaWt0**
Changes:
- remove Dockerfile
- add `db.env` to use by postgres image
- add `go get github.com/tomnomnom/linkheader` to `build_and_go.sh`

Fix issue #6 and #3 

So everything works w/ `docker-compose up`

One thing to note: maybe needed to remove old postgres container because of postgres superuser creation flow (creates on first build)

### CODE REVIEW

**Comment by vz10** [comment]
Maybe we have to create some initial Go script which will check availability of tables and create them if needed

**Comment by cmltaWt0** [comment]
I agree. Lets create issue for this.

**Comment by cmltaWt0** [comment]
@vz10 hmm, I forgot to change readme. Wait a minute -I will add commit for this.

File: README.md [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
 git clone https://github.com/vz10/geogit.git
+cd geogit
 ```
 
 If you use Kitematic then run
 ```
-# cd geogit
-# eval "$(docker-machine env default)"
-# docker build -t geogit .
+eval "$(docker-machine env default)"
```
Comment (vz10) [inline README.md:10]: Maybe you have to add something like "If you are an idiot without native docker"

**Comment by TamoshaytisV** [comment]
👍

**Pull request merged**